src/main/java/com/metsci/laproc/datareference/DataReferenceImpl.java

```diff
@@ -109,4 +129,17 @@ public void setAxisFunctionOnGraph(ParametricFunction xAxis, ParametricFunction
         notifyObservers();
     }
 
+    public void exportGraph(String filePath) throws IOException {
```

Comment:
Is there a reason that this function is in DataReferenceImpl? It doesn't seem like this behavior is related to what this class is doing.